tracer/test/Datadog.Trace.TestHelpers/MockAttributeArrayValue.cs

```diff
@@ -0,0 +1,28 @@
+// <copyright file="MockAttributeArrayValue.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using MessagePack;
+
+namespace Datadog.Trace.TestHelpers
```

Comment:
According to the other `TestHelpers`, wouldn't using a file scope namespace be more appropriate? 🙂
like so:
```Csharp
namespace Datadog.Trace.TestHelpers;
```

Reply:
So for this one, looking at the similar mock files on the location of it they use hence leaving it as it is ATM, but not opposed to change it if needed.